langtest/utils/custom_types/sample.py

```diff
@@ -677,7 +677,10 @@ def run(self, model, **kwargs):
         Returns:
             bool: True if the sample ran successfully, False otherwise.
         """
-        dataset_name = self.dataset_name.split("-")[0].lower()
+        dataset_name = (
+            self.dataset_name.split("-")[0].lower() if self.dataset_name else "sum_prompt"
+        )
+
```

Comment:
Does that refer to one of the key in `default_user_prompt`?? If yes, let's rename the key directly so that we don't have to handle edge cases

Reply:
@JulesBelveze 
right know we were acessing the `default_user_prompt` based on dataset that we have in `langtest`

But, huggingface contain so many dataset so we will be using `sum_prompt` for all the huggingface dataset where task is summarization.

This is not the bug , this is is already  in  pr - https://github.com/JohnSnowLabs/langtest/pull/589

but due to some conflict I think the changes are not updated in the release/1.1.0  branch

Reply:
Gotcha, could we rename it to something more self explanatory like "hf-dataset" or something? 😅

Reply:
@JulesBelveze 
This prompt is  like ` default summarization prompt `  not restricted to hf-dataset  only.
if you have any suggestion we can do that

Reply:
Let's go for `default-summarization-prompt`